examples/building.rs

```diff
@@ -230,3 +530,22 @@ impl Config for Game {
         });
     }
 }
+
+fn face_wall_block(face: BlockFace, normal: BlockState, wall: BlockState) -> BlockState {
+    match face {
+        BlockFace::Bottom => normal,
+        BlockFace::Top => normal,
+        BlockFace::North => wall.set(PropName::Facing, PropValue::North),
+        BlockFace::South => wall.set(PropName::Facing, PropValue::South),
+        BlockFace::West => wall.set(PropName::Facing, PropValue::West),
+        BlockFace::East => wall.set(PropName::Facing, PropValue::East),
+    }
+}
+
+fn face_directional_block(face: BlockFace, block: BlockState) -> BlockState {
+    match face {
+        BlockFace::Bottom | BlockFace::Top => block.set(PropName::Axis, PropValue::Y),
+        BlockFace::North | BlockFace::South => block.set(PropName::Axis, PropValue::Z),
+        BlockFace::West | BlockFace::East => block.set(PropName::Axis, PropValue::X),
+    }
+}
```

Comment:
I think these should definitely go in the library itself, with unit tests.

Reply:
Yeah, but where? Would it be best to put it inside `util.rs`? Or would it be better to put it inside `block.rs`?

Reply:
Lets put it in block.rs for now, we can always move it later.